### PR TITLE
Feature/bench 1119 add accordion

### PR DESCRIPTION
**Pull request by okyd-capco**
## Ticket

(https://ilabs-capco.atlassian.net/jira/software/c/projects/BENCH/boards/135?modal=detail&selectedIssue=BENCH-1157)

## Description

Create an accordion component and moved sub-content into them for a cleaner user experience.

## WIP

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

*PRs that should not be reviewed should be marked as a WIP.

- [ ] This PR is currently a work-in-progress.
- [x] This PR is ready for review.

## Checklist

- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors.
- [x] All existing and new tests are passing.
- [x] My PR meets the acceptance criteria as outlined by my ticket.
- [x] I updated/added relevant documentation and comments (docs start with `///` and comments with `//`).
- [x] Static analysis (e.g. Snyk) does not report any problems on my PR.

## Screenshots

*Screenshots can be added here.*

### CODE REVIEW

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              General settings
+            </Typography>
+            <Typography sx={{ color: "text.secondary" }}>
+              I am an accordion
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <Typography>
+              Nulla facilisi. Phasellus sollicitudin nulla et quam mattis
+              feugiat. Aliquam eget maximus est, id dignissim quam.
```
Comment (khwt-capco) [inline src/components/Accordion/Accordion.tsx:39]: Should there be hardcoded example text here or is this a mistake?

Comment (okyd-capco) [reply src/components/Accordion/Accordion.tsx:39]: This cord needs to be removed, I'll remove it now

File: src/components/Content/Content.tsx [hunk 2]
```diff
@@ -17,91 +22,72 @@ type ReferenceContentType = {
...
+                            'rel="noreferrer" target="_blank" href='
+                          ),
+                        }}
+                      />
+                    </Typography>
+                  </AccordionDetails>
+                </Accordion>
+              </Wrapper>
+              {/* <div>{name}</div>
             <div
```
Comment (tbrj-capco) [inline src/components/Content/Content.tsx:76]: Removed the commented code, if not using anymore

**Review by tbrj-capco — COMMENTED** [review]
Screenshot 2022-03-10 at 11 30 53
Getting this compilation warning

**Comment by okyd-capco** [comment]
compilation warning has been resolved @tbrj-capco

**Review by crmv-capco — APPROVED** [review]

**Pull request merged**